### PR TITLE
Initial documentation

### PR DESCRIPTION
**Pull request by AJenbo**
This simply describes the over all steps of installing and configuring Tiled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Diablo 1 modding tools
+
+This is a set of tools for modding Diablo 1, and more specificually DevilutionX.
```
Comment (mewmew) [inline README.md:3]:
```suggestion
This is a set of tools for modding Diablo 1, and more specifically DevilutionX.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+## Map Making
+
+For level editing we build on top of [Tiled](https://www.mapeditor.org/), a versitile tilebased level editor with many features that can gratly speed up the process of making a level. Please see the [mapping section](Tiled/README.md) for how to set it up, and use it effectivly.
```
Comment (mewmew) [inline README.md]:
```suggestion
For level editing we build on top of [Tiled](https://www.mapeditor.org/), a versatile tile-based level editor with many features that can greatly speed up the process of making a level. Please see the [mapping section](Tiled/README.md) for how to set it up, and use it effectively.
```

File: Tiled/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Diablo 1 level editing with Tiled
+
+We provide an extension that allows [Tiled](https://www.mapeditor.org/) to open and save to the dun-fileformat used by Diablo. You will also fine a series of prefab stamps, as well as an extensive set of rules for automating parts of the process of creating an interesting and detailed level.
```
Comment (mewmew) [inline Tiled/README.md]:
```suggestion
We provide an extension that allows [Tiled](https://www.mapeditor.org/) to open and save to the DUN-file format used by Diablo. You will also find a series of prefabricated stamps, as well as an extensive set of rules for automating parts of the process of creating an interesting and detailed level.
```

File: Tiled/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+![Diablo map in Tiled](Tiled/docs/images/tiled.png)
+
+## Setting up Tiled
+
+### Installation
+
+First step is to download and install [Tiled](https://www.mapeditor.org/), we recomend version 1.4 as it makes setting up the tool a lot simpler, but 1.3 should also work.
```
Comment (mewmew) [inline Tiled/README.md]:
```suggestion
First step is to download and install [Tiled](https://www.mapeditor.org/), we recommend version 1.4 or higher as it makes setting up the tool a lot simpler, but 1.3 should also work.
```

File: Tiled/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Then download [this repository](https://github.com/diasurgical/modding-tools/archive/master.zip).
+
+Next you will need a PNG version of the tileset, this can be done using [Open source-ami](https://github.com/sanctuary/opensource-ami).
+Note for Windows users we provide a helper script in the windows-x86-scripts folder, this need to be used togeather with Open source-ami.
```
Comment (mewmew) [inline Tiled/README.md]:
```suggestion
Note for Windows users we provide a helper script in the [windows-x86-scripts folder](https://github.com/diasurgical/modding-tools/tree/master/windows-x86-scripts), this need to be used together with Open source-ami.
```

File: Tiled/README.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+Once you have at the, this should give you 5 png-files that needs to be placed in the Tiled/tilesets of this repository's files.
```
Comment (mewmew) [inline Tiled/README.md]:
```suggestion
Once you have run the script, this should give you 5 PNG-files that needs to be placed in the Tiled/tilesets directory of this repository's files.
```

Comment (mewmew) [reply Tiled/README.md]: Consider adding comment about `objects.png` and `monsters.png`.

File: Tiled/README.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    Tiled/tilesets/cathedral_mega_special.png
+    Tiled/tilesets/caves_mega.png
+    Tiled/tilesets/hell_mega.png
+    Tiled/tilesets/monsters.png
+    Tiled/tilesets/objects.png
+    Tiled/tilesets/tristram_mega_special.png
+
+### Setting up tiled
+
+From the menu select Project->Project Properties, here you need to point Tiled to the `extensions` folder and the `rules.txt` file:
```
Comment (mewmew) [inline Tiled/README.md]:
```suggestion
From the menu select `Project->Project Properties`, here you need to point Tiled to the `extensions` folder and the `rules.txt` file:
```

**Review by mewmew — APPROVED** [review]
LGTM, with minor comments. We can flesh out the documentation later if we notice there are parts where users get stuck.

**Comment by AJenbo** [comment]
@mewmew correction and suggestions applied, plus added the part about grid snapping.
I also adjusted the windows tile sheet generation documentation, so hopefully it should now be enough to get people up and running with the tools.

**Pull request merged**